css/style.css

```diff
@@ -17,3 +17,388 @@ body {
  */
 
 
+.logo {
+    width: 2em;
+    height: auto;
+    padding-top: 2em;
+    padding-left: 2em;
+}
+
+header {
+    height: 6em;
+    padding-left: 6em;
+    background-color: rgba(196, 196, 196, 0.1);
+}
+
+nav {
+    float: right;
```

Comment:
Lucy said, 'float' property is not very much in trend anymore. so, may be not use it